### PR TITLE
Use country template to run core tests

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to #492 
Depends on #505 
Depends on #508 
Depends on openfisca/country-template#4

### CODE REVIEW

File: openfisca_core/formulas.py [hunk 1]
```diff
@@ -739,7 +739,10 @@ def new_filled_column(
     assert issubclass(formula_class, AbstractFormula), formula_class
 
     if definition_period is UnboundLocalError:
-        raise ValueError(u'definition_period missing in {}'.format(name).encode('utf-8'))
+        if reference_column:
```
Comment (fpagnoux) [inline openfisca_core/formulas.py]: Should be done in a separate PR

**Comment by cbenz** [comment]
The commit 0c8131cba920505d2afa4049ae712c2430a0dd66 makes the tests fail for me. Without it, the tests pass.

**Comment by fpagnoux** [comment]
Arg cette PR n'a pas été mergée dans la bonne branche !!!
